__tests__/app.test.js

```diff
@@ -45,6 +45,7 @@ describe("getArticleById", () => {
           expect(article.body).toEqual(expect.any(String));
           expect(article.created_at).toEqual(expect.any(String));
           expect(article.votes).toEqual(expect.any(Number));
+          expect(article.comment_count).toEqual(expect.any(Number));
```

Comment:
we shouldn't have to refactor old tests for this if we are it means the test was brittle